mypy/fastparse.py

```diff
@@ -27,7 +27,7 @@
 from mypy.errors import Errors
 
 try:
-    from typed_ast import ast35
+    from typed_ast import ast3  # type: ignore
```

Comment:
I'm guessing the typeshed stubs for typed_ast need an update? I can work on that if you're not already doing it.